.github/workflows/perf.yml

```diff
@@ -39,7 +39,12 @@ jobs:
       - uses: wyvox/action-setup-pnpm@v3
         if: steps.did-change.outputs.changed == 'true'
         with:
-          node-version: '20.1.0'
+          node-version: '20.18.0'
+
+      - uses: browser-actions/setup-chrome@v1
+        with:
+          chrome-version: 128
```

Comment:
I think puppeteer manages it's own chrome version.
You might need to downgrade puppeteer instead.
It had an update 4 days ago. Maybe its related

Reply:
I now saw that there is also puppeteer-chromium-resolver. According to docs it should be able to detect local chrome. But i think chrome must be installed before doing pnpm install.